CONTRIBUTING.md

```diff
@@ -21,3 +21,32 @@ Luckily, if you're not willing to get too involved with the technicalities of Gi
 The [Article Styling Criteria (ASC)](https://github.com/ppy/osu-wiki/tree/master/wiki/Article_styling_criteria) is a set of standards that must be adhered to for all new and edited articles going forward. Among these are rules regarding project structure, usage of Markdown, formatting, diction and grammar. Please refer to it if you have any specific questions or concerns.
 
 The ASC is fairly verbose and encompasses many minute details, that are not necessary to memorise in order to take the first step of contributing to the osu! wiki. Have a look at the [contribution guide](https://github.com/ppy/osu-wiki/tree/master/wiki/osu!_wiki/Contribution_guide), and other contributors will also be happy to answer your questions in the `#osu-wiki` channel of the [osu!dev Discord server](https://discord.gg/ppy).
+
+## Testing
+
+All pull requests are automatically run through a series of continuous integration checks (CI) in order to catch various common errors.
+
+For people familiar with the command line, these CI checks can also be run locally.
+
+### Requirements
+
+- Python 3.X
+- Node.js 16
+
+### Setup
+
+```sh
+pip3 install venv
+python3 -m venv ./.venv
+source ./.venv/bin/activate
+pip install -r requirements.txt
```

Comment:
```suggestion
pip3 install -r requirements.txt
```

Although I argue the value of that txt file. Why not just include it inline?

Reply:
we can inline it as it only consists of a single requirement for now, but I'd prefer reusing it here and in the GitHub workflow for unification purposes and as it may potentially grow like node's `package.json` (hopefully not to that extent though)